views/index.html

```diff
@@ -19,24 +19,28 @@ <h1 class="text-center">Tic Tac Toe</h1>
 
   <div id = "game-view" class='container board text-center'>
     <div id = "game-view-info">
-      <div class="scores"></div>
-        <div class="result"></div>
-        <button class="btn btn-dark" id = "restart">Restart Game</button>
+      <div class="row">
+          <div class="col-lg-4"><div class="scores"></div></div>
+          <div class="col-lg-4"><div class="result"></div></div>
+          <div class="col-lg-4"><button class="btn btn-transparent" id = "restart">Restart</button></div>
```

Comment:
give this border or background, does not appear like a button on screen.
![image](https://user-images.githubusercontent.com/5979667/68602475-673b4b00-04cc-11ea-845f-b28c6b32983c.png)